src/components/TaskCharts.js

```diff
@@ -0,0 +1,105 @@
+import _ from 'lodash';
+import { Component, PropTypes } from 'react';
+import Task from './Task';
+import { createTaskChart } from './../charts/createTaskChart';
+import rd3 from 'react-d3-library';
+
+/**
+Notes:
+Adjusted inline CSS to match design specs, which results in non centered visualizations.  Can adjust.
+**/
+
+/**
+Import React Components from RD3 library to convert D3 into React Elements,
+otherwise virtual dom will be effected negatively, and diffing algo won't work.
+**/
+
+const CompleteComponent = rd3.Component;
+const InProgressComponent = rd3.Component;
+const ToDoComponent = rd3.Component;
+
+class TaskCharts extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {completeData: '', inProgressData: '', toDoData: ''};
```

Comment:
Ah, Okay I deleted my previous message thinking I found a solution.  I played with the TaskCharts a bit, if I were to replace the library the rendering would not be an issue.  Due to the way the react-d3-library is set up, it's expecting data to update, so I would probably need a new action in Redux that would create the charts, and use componentDidMount to update the changes without triggering a new state in the same component.

I would say there aren't any advantages comparatively to Redux because it creates immutable states.  Hmm however with React's diffing algorithm, the changes are very small in the overall sense of things, but yes I think there are no advantages to the way it's currently set up.